README.md

```diff
@@ -1,63 +1,101 @@
-# Maslo Companion App
+# Maslo Companion Kit
 
 
 Clone the respository and ensure you have the requirements below.
 
 ### Requirements
 
+ * React Native CLI development environment for both iOS and Android (instructions [`here`](https://reactnative.dev/docs/environment-setup))
+
  * Node.js 10 ([`nvm`](https://github.com/nvm-sh/nvm) is preferable)
 
  To install:
  ```nvm install 10```
- 
+
  To switch to version 10:
  ```nvm use 10```
+
  * Expo CLI
  ```
  npm i -g expo-cli
  ```
+ * React-native CLI
+ ```
+ npm i -g react-native-cli
+ ```
  * Firebase Tools:
  ```
  npm i -g firebase-tools
  ```
```

Comment:
Suggest reformatting to keep things a bit more tidy and have the same format as the Node instructions for all items:
```suggestion
 To switch to version 10:  ```nvm use 10```

 * Expo CLI  
 To install: ```npm i -g expo-cli```
 
 * React-native CLI  
 To install: ```npm i -g react-native-cli```
 
 * Firebase Tools:  
 To install: ```npm i -g firebase-tools```
```